initial-balances-generator-etherscan.js

```diff
@@ -12,10 +12,16 @@ const args = require('minimist')(process.argv.slice(2));
 //const toBlock   = 6772802
 
 // Phase 1
-const url1 = 'https://etherscan.io/tx/0x6dd35f57ac61436c9ee7ebc2cd0be830506965aca072eb23dc0e4e4b6456cbfb';
-const url2 = 'https://etherscan.io/tx/0xed2415a9e8026e042c58e701bfbe218f858ad71bfa9796c9b8936b8faae83155';
-const fromBlock = 6772803; // included
-const toBlock   = 7225342 // included
+//const url1 = 'https://etherscan.io/tx/0x6dd35f57ac61436c9ee7ebc2cd0be830506965aca072eb23dc0e4e4b6456cbfb';
+//const url2 = 'https://etherscan.io/tx/0xed2415a9e8026e042c58e701bfbe218f858ad71bfa9796c9b8936b8faae83155';
+//const fromBlock = 6772803; // included
+//const toBlock   = 7225342 // included
+
+// Phase 2
+const url1 = 'https://etherscan.io/tx/unknown';
+const url2 = 'https://etherscan.io/tx/unknown';
+const fromBlock = 7225342; // included
```

Comment:
Is this block included in two phases?

Reply:
I don't know, I did just copy the values from @nikita-fuchs other script

Reply:
So if a relevant tx is in this block, then that tx may be counted twice i.e. in phase 1 and phase 2?

Reply:
I assume so, I will adjust it to use the following block

Reply:
`7225342` or `7225343` yields in the same accounts.json hash, so 7225342 did not include any migrations. `7225343` is the more correct one as it is the first one to be included in phase 2.